frontend/src/components/search/SearchCard.jsx

```diff
@@ -0,0 +1,19 @@
+// NPM package
+import { useRecoilValue } from "recoil";
+import { Link } from "react-router-dom";
+
+// Project files
+import { allUsers } from "../../state/usersData";
+import UserApi from "../../api/UserApi";
+import Profile from "../../components/profile/Profile";
+
+export default function SearchCard({ match }){
+    // Global state
+    const profile = useRecoilValue(allUsers);
```

Comment:
Please could we name this `profiles` (plural) so it's clearer it is all the users